README.md

```diff
@@ -1,7 +1,32 @@
 # Tom's News API
 
-To connect to the development and test databases locally, create two files named '.env.test' and '.env.development'. In each file, declare the name of the database you would like to connect to in the form: 'PGDATABASE=' followed by the database name. You will want to use '.env.test' to connect to a database of test data if applicable.
+Hosted App: https://its-the-news-2.onrender.com
 
-To create the databases, use 'npm install' in the terminal followed by 'npm run setup-dbs'.
+## Project Summary 
+This project is planning to be a basic news app, containing articles for the user to browse and interact with. Features include being able to post comments on certain articles, increment a vote count for each article and search for articles by topic. 
```

Comment:
This description sounds more like it's describing a front end website - this is just the API that allows a user to get article data from the database and post, delete, etc.